packages/design-picker/src/components/premium-badge/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Gridicon } from '@automattic/components';
+import { Tooltip } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import classNames from 'classnames';
+import type { FunctionComponent } from 'react';
+
+import './style.scss';
+
+interface Props {
+	className?: string;
+}
+
+const PremiumBadge: FunctionComponent< Props > = ( { className } ) => {
+	const { __ } = useI18n();
+
+	return (
+		<Tooltip
+			position="top center"
+			// @ts-expect-error: @types/wordpress__components doesn't align with latest @wordpress/components
+			delay={ 300 }
+			text={ `${ __(
+				'Let your site stand out from the crowd with a modern and stylish Premium theme.',
```

Comment:
@arthur791004 Thanks for thinking ahead on this and making these two separate i18n strings! 😄

Reply:
Is this string 'Let your site stand out from the crowd with a modern and stylish Premium theme.', already translated?    If so, then don't let the new string block merging this PR.

 We can add the new string in a subsequent PR. (I think translations can take a week or two right? )

Reply:
I see the first string is done: https://translate.wordpress.com/developer/arthur791004 😄

Reply:
Okay, I'll add a new string in follow-up PR.